performance_metrics/src/stat_logger.cpp

```diff
@@ -94,17 +94,17 @@ void log_trackers_latency_all_stats(
       stream << header_title << std::endl;
       stream_out(csv_out, stream, "node", wide_space);
       stream_out(csv_out, stream, "topic", wide_space);
-      stream_out(csv_out, stream, "size[b]", narrow_space);
-      stream_out(csv_out, stream, "received[#]", wide_space);
-      stream_out(csv_out, stream, "late[#]", narrow_space);
-      stream_out(csv_out, stream, "too_late[#]", wide_space);
-      stream_out(csv_out, stream, "lost[#]", narrow_space);
-      stream_out(csv_out, stream, "mean[us]", narrow_space);
-      stream_out(csv_out, stream, "sd[us]", narrow_space);
-      stream_out(csv_out, stream, "min[us]", narrow_space);
-      stream_out(csv_out, stream, "max[us]", narrow_space);
-      stream_out(csv_out, stream, "freq[hz]", narrow_space);
-      stream_out(csv_out, stream, "throughput[Kb/s]", wide_space, false);
+      stream_out(csv_out, stream, "size_b", narrow_space);
+      stream_out(csv_out, stream, "received_num", wide_space);
```

Comment:
I'd replace `received_num` with `received_msgs` (and same with othes that use `_num`, like `late_num` -> `late_msgs`

Reply:
ok cool will do, thanks @mauropasse